.gitignore

```diff
@@ -2,3 +2,4 @@ vendor
 **/_build
 dist/
 .idea
+.generated-yaml/
```

Comment:
why this change?

Reply:
When testing it locally, I noticed that this directory existed and was added to my commit. I think it's not needed anymore after we replaced shell scripts with Go, but there might be workspaces out there with it.